middleware/auth/cognitoJwtAuth.go

```diff
@@ -54,7 +53,7 @@ func (middleware CognitoJwtAuthMiddleware) Execute(next http.Handler) http.Handl
 			return
 		}
 
-		r = r.WithContext(context.WithValue(r.Context(), request.UserIDContextKey, cognitoClaims.Subject))
+		r = r.WithContext(request.WithUserID(r.Context(), cognitoClaims.Subject))
```

Comment:
FYI: It's usually considered "bad form" to expose your context keys outside the package and instead preferred to create helper methods that do it for you.

Reply:
It's kind of one of those "don't expose the interworkings on your packages if you can avoid it" things. https://medium.com/@matryer/context-keys-in-go-5312346a868d covers it a bit.